internal/sub_reconcilers/reconcile_namespace.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright The L5 Operator Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sub_reconcilers
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	petsv1 "github.com/opdev/l5-operator-demo/api/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	namespaceLabel = "openshift.io/cluster-monitoring"
+)
+
+type NamespaceReconciler struct {
+	client client.Client
+	Log    logr.Logger
+	Scheme *runtime.Scheme
+}
+
+func NewNamespaceReconciler(client client.Client, log logr.Logger, scheme *runtime.Scheme) *NamespaceReconciler {
+	return &NamespaceReconciler{
+		client: client,
+		Log:    log,
+		Scheme: scheme,
+	}
+}
+
+func (r *NamespaceReconciler) Reconcile(ctx context.Context, bestie *petsv1.Bestie) (ctrl.Result, error) {
+	logInfo := types.NamespacedName{
+		Namespace: bestie.Namespace,
+		Name:      "NamespaceReconciler",
+	}
+	r.Log.WithValues("Namespace", logInfo)
```

Comment:
Should probably say NamespaceReconciler instead of Namespace as it may be confusing

Reply:
mmm actually this is more of a label reconciler no ?

Reply:
Yep, you're right. Thank you for catching this, I was probably thinking too much on namespace when naming it.